packages/orbit-components/src/Box/helpers/getWrap.ts

```diff
@@ -2,7 +2,7 @@ import type { Props } from "../types";
 import type { QUERIES } from "../../utils/mediaQuery/consts";
 import { wrapClasses } from "../../common/tailwind/wrap";
 
-const getWrapClass = (wrap: Props["wrap"], viewport?: QUERIES) => {
+const getWrapClass = (wrap?: Props["wrap"], viewport?: QUERIES): string | null => {
```

Comment:
`Props["wrap"]` already includes `undefined` so isn't the `?` redundant here?

The same for all other files where the `?` was added

Reply:
Yep, forgot about that, but it's less obvious if it's required or not without it :)